unstructured/ingest/cli/cmds/utils.py

```diff
@@ -0,0 +1,48 @@
+from gettext import gettext as _
+
+import click
+
+
+class Group(click.Group):
+    def parse_args(self, ctx, args):
+        try:
+            return super().parse_args(ctx, args)
+        except click.MissingParameter:
+            if "--help" not in args:
+                raise
+
+            # remove the required params so that help can display
+            for param in self.params:
+                param.required = False
+            return super().parse_args(ctx, args)
+
+    def format_commands(self, ctx: click.Context, formatter: click.HelpFormatter) -> None:
+        """Extra format methods for multi methods that adds all the commands
+        after the options.
+        """
+        commands = []
+        for subcommand in self.list_commands(ctx):
+            cmd = self.get_command(ctx, subcommand)
+            # What is this, the tool lied about a command.  Ignore it
+            if cmd is None:
+                continue
+            if cmd.hidden:
+                continue
+
+            commands.append((subcommand, cmd))
+
+        # allow for 3 times the default spacing
+        if len(commands):
+            if formatter.width:
+                limit = formatter.width - 6 - max(len(cmd[0]) for cmd in commands)
+            else:
+                limit = -6 - max(len(cmd[0]) for cmd in commands)
+
+            rows = []
+            for subcommand, cmd in commands:
+                help = cmd.get_short_help_str(limit)
+                rows.append((subcommand, help))
+
+            if rows:
+                with formatter.section(_("Destinations")):
+                    formatter.write_dl(rows)
```

Comment:
what's going on here?

Reply:
This is a copy of the existing `format_commands` method on the `Group` class in click but replaces `Commands` -> `Destinations` to make is more obvious that the subcommands are related to downstream connectors.

Reply:
got it. is this potentially brittle to updates in click?

Reply:
It's still using the same logic so only changes to this _exact_ method wouldn't be reflected, but this it really just used for formatting so I don't seen this being high risk.